src/traccuracy/utils.py

```diff
@@ -0,0 +1,37 @@
+import copy
+
+import numpy as np
+
+from traccuracy import NodeFlag, TrackingGraph
+from traccuracy.matchers import Matched
```

Comment:
I tried to put this under type checking but then tests failed. I don't understand why this isn't considered purely a typing use...

Reply:
@cmalinmayor can probably double check me on this, but would adding `from __future__ import annotations` allow you put those imports under type checking?